proto/profile.proto

```diff
@@ -115,6 +120,36 @@ message Sample {
   // label includes additional context for this sample. It can include
   // things like a thread id, allocation size, etc
   repeated Label label = 3;
+  // Breakdown can hold the individual timestamps and values for a sample that
+  // has aggregated values. The type and unit of each breakdown is defined by
+  // the corresponding entry in Profile.sample_type. It's acceptable to have
+  // less, but not more, breakdown entries than Profile.sample_type entries.
+  repeated Breakdown breakdown = 4;
+}
+
+// Breakdown stores data about the individual events that have been aggregated
+// into a Sample. There can be one breakdown for each Profile.sample_type.
+//
+// TODO: Figure out if this can also be used for the exemplar UC from alexey:
+// https://github.com/google/pprof/pull/725#issuecomment-1264841904
+message Breakdown {
+  // A list of timestamps at which the values of the associated sample were
+  // collected. The time is given in Profile.tick_unit resolution relative to
+  // Profile.time_nanos and values are delta encoded. So for three events
+  // captured at 100, 110 and 150 ticks after the profile start, this list should
+  // hold the values [100, 10, 40].
+  //
+  // TODO: Is delta encoding going to safe enough space to justify the
+  // complexity?
+  repeated int64 ticks = 1;
+  // A list that is either empty or holding exactly the same number of elements
+  // as Breakdown.ticks, associating each point in time with a recorded value.
```

Comment:
> A list that is either empty or holding exactly the same number of elements as Breakdown.ticks

I find this wording somewhat confusing. How the profile with non empty `ticks` and empty `values` should be interpreted?

Also, I think that comment for `Breakdown.label_set_id` could be simplified to just say that it need to have the same number of elements as `Breakdown.values`

Reply:
> I find this wording somewhat confusing. How the profile with non empty ticks and empty values should be interpreted?

I'll clarify this. The idea is that all values are the same and can be calculated by `Sample.value/len(Breakdown.ticks)`.